docs/usage/dataclasses.md

```diff
@@ -365,7 +365,8 @@ class User:
 
 
 user = User(id='42')
-print(json.dumps(user, indent=4, default=pydantic_encoder))
+# TODO replace this with methods or equivalent
+print(pydantic_core.to_json(user, indent=4).decode())
```

Comment:
Remind me to ask you how the pydantic_core to_json function works

Reply:
will do.